2.13-RUN-9843 (#347)
run-ai/docs: docs/Researcher/user-interface/workspaces/create/create-env.md

To create an environment:

1. In the left menu, press *New Environment*.

> 2. In the *Scope* pane, choose one or more items from the tree. The environment is assigned to one or more projects in the scope.

Reviewer: ```suggestion
2. In the *Scope* pane, choose one item from the tree. The compute resource is assigned to that item and all its subsidiaries.
```
